packages/utils/src/lib/mock/schema-helper.mock.ts

```diff
@@ -13,7 +13,8 @@ const __pluginSlug__ = 'mock-plugin-slug';
 const __auditSlug__ = 'mock-audit-slug';
 const __groupSlug__ = 'mock-group-slug';
 const __categorySlug__ = 'mock-category-slug';
-const __outputPath__ = 'tmp/out-execute-plugin.json';
+const __outputDir__ = 'tmp';
+const __outputFile__ = 'out-execute-plugin.json';
```

Comment:
All files inside `tmp`, please :pray: 

```suggestion
const __outputFile__ = `${__outputDir__}/out-execute-plugin.json`;
```